From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Byteflux <byte@byteflux.net>
Date: Mon, 6 Jan 2020 17:59:10 -0800
Subject: [PATCH] Add configuration API


diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 7c0a788900c93c29d14d8c45ac5ae3317cf4a94e..2d4e5ad79d7300f89c65457a9db7306d15bc7b94 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1456,6 +1456,14 @@ public interface Server extends PluginMessageRecipient {
         }
         // Paper end
 
+        // Nozzle start
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getNozzleConfig()
+        {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Nozzle end
+
         /**
          * Sends the component to the player
          *
